### PR TITLE
Docs: Readme, Contribution Guide, Code of Conduct

### PR DESCRIPTION
**Pull request by mymphe**
- Update readme
- add Contribution Guide
- add Code of Conduct

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,86 @@
...
-- hardhat
-- ethers v6
-- typechain
-- commitlint
-- lint-staged
-- eslint
-- prettier
-- solhint
-- slither
+<div>
```
Comment (tamtamchik) [inline README.md:15]: Why not markdown syntax?

```
[![Lido V2][ico-version]][link-version]

... somewhere at the bottom of the readme ...

[ico-version]: https://img.shields.io/badge/v2-version?label=lido&labelColor=rgb(91%2C%20162%2C%20252)&color=white
[link-version]: https://lido.fi
```

This is opinionated, but this way, it is more readable than embedded HTML.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+- **Bug Reports**. If you encounter a bug, please report it using the GitHub issues feature. Check first to ensure the bug hasn't already been reported. If it has, you can contribute further by adding more detail to the existing report. _Note that this only relates to off-chain code (tests, scripts, etc.), for bugs in contracts and protocol vulnerabilities, please refer to [Bug Bounty](/README.md#bug-bounty)_.
+
+- **Feature Requests**: Have an idea for a new feature or an improvement to an existing one? Submit a feature request through the GitHub issues, detailing your proposed enhancements and how they would benefit the Lido Finance Core.
+
+### Improving Documentation
+
+Good documentation is crucial for any project. If you have suggestions for improving the documentation, or if you've noticed an omission or error, making these corrections is a significant contribution. Whether it's a typo, additional examples, or clearer explanations, your help in making the documentation more accessible and understandable is highly appreciated.
+
+For expansive documentation, visit the [Lido Docs repo](https://github.com/lidofinance/core).
```
Comment (tamtamchik) [inline CONTRIBUTING.md:19]: https://github.com/lidofinance/docs?

**Review by tamtamchik — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,40 +1,86 @@
...
+    <img alt="Lido" src="https://img.shields.io/badge/v2-version?label=lido&labelColor=rgb(91%2C%20162%2C%20252)&color=white"/>
```
[the remark this answers is not in the document]
Comment (mymphe) [reply README.md:16]: Yeah, this seems to be a default behaviour in readmes. All images are wrapped in anchor tags which lead to their sources.

File: README.md (hunk 1 above)
Comment (mymphe) [reply README.md:15]: I'm not sure it's more readable though. It's more consistent I agree but with HTML everything is in one place without cryptic references. Also we used html for styling and positioning.

**Pull request merged**